Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -261,25 +204,7 @@ public class XPCMachServer {
         self.routesWithMessageWithReply[route.route] = handlerWrapper
     }
     
-    /// Begins processing requests received by this XPC server.
-    ///
-    /// This function replicates [`xpc_main()`](https://developer.apple.com/documentation/xpc/1505740-xpc_main) behavior by never
-    /// returning.
-    public func start() -> Never {
-        // Start listener for the mach service, all received events should be for incoming connections
-        xpc_connection_set_event_handler(self.machService, { connection in
-            // Listen for events (messages or errors) coming from this connection
-            xpc_connection_set_event_handler(connection, { event in
-                self.handleEvent(connection: connection, event: event)
-            })
-            xpc_connection_resume(connection)
-        })
-        xpc_connection_resume(self.machService)
-        
-        dispatchMain()
-    }
-    
-    private func handleEvent(connection: xpc_connection_t, event: xpc_object_t) {
+    internal func handleEvent(connection: xpc_connection_t, event: xpc_object_t) {
```

Comment:
Is there a reason this shouldn't remain `private`?

Reply:
It's called from both subclass implementations of `start()`.

Reply:
Ah whoops, not sure I how I forgot that.